### DISCUSSION TITLE
Do We Have Support for SSO in This Plugin?

### DISCUSSION THREAD

**Discussion by Hamzah-syed** [AxeWP/wp-graphql-headless-login#148] [Q&A] [answered]
I'm currently using the WP-GraphQL-Headless-Login plugin and was wondering if it has support for Single Sign-On (SSO) or if there are any helper functions available for SSO integration. I couldn't find any direct documentation on this, so any guidance would be really appreciated.

If SSO support isn’t included, does anyone have recommendations or steps on how to achieve SSO? I’d love to know if there are workarounds, custom functions, or other strategies that have worked for others. Any help or insights are welcome!

Thanks in advance!

**Comment by justlevine** [comment, marked as answer]
Hey @Hamzah-syed can you clarify what you mean by `SSO` in this context? There are a handful of bundled OpenID providers + a generic OAuth2 provider for other configs.

If you're looking for a different flavor of SSO than OAuth2, you can [create a custom `ProviderConfig`](https://github.com/AxeWP/wp-graphql-headless-login/blob/develop/docs/recipes/provider-config.md)
